calico/_plugins/helm.rb

```diff
@@ -29,7 +29,8 @@ def initialize(tag_name, extra_args, liquid_options)
       # iterating over files in that directory.
       extra_args.gsub!(/--execute-dir (\S*)/) do |_|
         e = []
-        Dir.foreach "_includes/charts/#{@chart}/#{$1}" do |file|
+        all_files = Dir.entries "_includes/charts/#{@chart}/#{$1}"
+        all_files.sort.each do |file|
             fpath = File.join($1, file)
```

Comment:
We need to sort here, because the ordering from Dir.entries is platform dependent.